salt/metalk8s/beacon/certificates.sls

```diff
@@ -0,0 +1,47 @@
+{%- from "metalk8s/macro.sls" import pkg_installed with context %}
+{%- from "metalk8s/map.jinja" import certificates with context %}
+
+include:
+  - metalk8s.repo
+
+Install pyOpenSSL:
+  {{ pkg_installed('pyOpenSSL') }}
+    - require:
+      - test: Repositories configured
+
+{%- if certificates.pem %}
+Add beacon for certificates expiration:
+  beacon.present:
+    - name: watch_certificates_expiry
+    - save: True
+    - beacon_module: cert_info
+    - interval: {{ certificates.beacon.interval }}
+    - disable_during_state_run: True
+    - notify_days: {{ certificates.beacon.notify_days }}
+    - files: {{
+        certificates.pem.values() | map(attribute='path') | list | json }}
+{%- endif %}
+
+{%- if certificates.kubeconfig %}
+  {%- set files = [] %}
+  {%- for kubeconfig in certificates.kubeconfig.values() %}
+    {%- do files.append({
+      "name": kubeconfig['path'],
+      "user": kubeconfig["cn"],
+    }) %}
```

Comment:
It's checked because it's the case in the function checking for the kubeconfig validity, so I followed what was done.
I wasn't sure if I should do it or not, to me it could be optional or we could retrieve it from the passed kubeconfig and so it would always match.

Reply:
To me, it should be optional and not checked by the beacon

Reply:
It's now optional and not checked by the beacon. :)